### DISCUSSION TITLE
Is there a way to use Grammarly with Quarto?

### DISCUSSION THREAD

**Discussion by kendonB** [quarto-dev/quarto-cli#3201] [Q&A]
Has anyone figured out how to efficiently use Grammarly with Quarto?

I presume the path of least resistance is to edit in a browser somehow. This would also likely enable other browser-based writing assistant tools (like wordtune) to work

**Comment by jmbuhr** [comment]
There is an unofficial language server: https://github.com/znck/grammarly
For VSCode is comes packaged in an extension (linked in the repo)

For neovim I installed it with `Mason` and configured via `require'lspconfig'.grammarly.setup{}` (https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#grammarly):

![image](https://user-images.githubusercontent.com/17450586/200294945-f3a6f7a1-ddd7-4122-9154-616220910996.png)

This is what it ends up looking like with my config (https://github.com/jmbuhr/.dotfiles/tree/main/nvim/.config/nvim)
